label_studio/tasks/api.py

```diff
@@ -223,20 +225,21 @@ def get_serializer_class(self):
         else:
             return TaskSimpleSerializer
 
-    def retrieve(self, request, *args, **kwargs):
-        task = self.get_object()
-        project = task.project
+    # def retrieve(self, request, *args, **kwargs):
+    #     print("def retrieve(self, request, *args, **kwargs):")
+    #     task = self.get_object()
+    #     project = task.project
 
-        # call machine learning api and format response
-        if project.evaluate_predictions_automatically:
-            for ml_backend in task.project.ml_backends.all():
-                ml_backend.predict_tasks([task])
+    #     # call machine learning api and format response
+    #     if project.retrieve_predictions_automatically:
+    #         for ml_backend in task.project.ml_backends.all():
+    #             ml_backend.predict_tasks([task])
 
-        result = self.get_serializer(task).data
+    #     result = self.get_serializer(task).data
 
-        # use proxy inlining to task data (for credential access)
-        result['data'] = task.resolve_uri(result['data'], project)
-        return Response(result)
+    #     # use proxy inlining to task data (for credential access)
+    #     result['data'] = task.resolve_uri(result['data'], project)
+    #     return Response(result)
```

Comment:
why is this commented out? If it's no longer needed, let's delete it. But we should understand what it was doing before